src/test/java/nextstep/subway/acceptance/LineAcceptanceTest.java

```diff
@@ -0,0 +1,200 @@
+package nextstep.subway.acceptance;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.testsupport.AcceptanceTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("지하철노선 관련 기능")
+public class LineAcceptanceTest extends AcceptanceTest {
+    private long upStationId;
+    private long downStationId;
+
+    @BeforeEach
+    public void setUp() {
+        upStationId = 지하철역_생성_요청("기흥역").jsonPath().getLong("id");
+        downStationId = 지하철역_생성_요청("신갈역").jsonPath().getLong("id");
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철노선 생성한다.")
+    @Test
+    void createStationLine() {
+        // given
+        지하철역_생성_요청("기흥역").jsonPath().getLong("id");
```

Comment:
```suggestion
        지하철역_생성_요청("기흥역");
```

`getLong` 을 할 필요가 있을까요?

Reply:
앗 그러네요 수정했습니다 ~